geoips/interfaces/base.py

```diff
@@ -472,7 +475,11 @@ def get_plugin(self, name):
     def get_plugins(self):
         """Retrieve a plugin by name."""
         plugins = []
-        for name in self._unvalidated_plugins.keys():
+        if not self._unvalidated_plugins:
+            raise PluginRegistryError(
+                "Plugin registry not found, please run 'create_plugin_registry'"
```

Comment:
Like other comments, should this be plural? "Plugin registries not found, please..."

Reply:
Yup. Agreed!

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/240/files/#r1373531491
Resolved there!